{"name": "test_001_panic_cancelled_by_driver", "status": "broken", "statusDetails": {"message": "TypeError: step() missing 1 required positional argument: 'string'", "trace": "pre_conditions = None\n\n>   ???\nE   TypeError: step() missing 1 required positional argument: 'string'\n\nC:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Scripts\\Xirgo_KB1_Testing\\03_Panic\\test_001_panic_cancelled_by_driver.py:30: TypeError"}, "attachments": [{"name": "stdout", "source": "62daf140-c70f-4ae8-936c-bd2f6f39d09c-attachment.txt", "type": "text/plain"}], "start": 1637785576168, "stop": 1637785576169, "uuid": "e7cd52a4-ad19-4f43-98da-1486a96d22fd", "historyId": "02901665ff7d29b5fc8edcc24ea152c7", "testCaseId": "e2d2586b80ab3718e96851129a3fbe89", "fullName": "Xirgo_KB1_Scripts.Panic.test_001_panic_cancelled_by_driver#test_001_panic_cancelled_by_driver", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Xirgo_KB1_Scripts.Panic"}, {"name": "suite", "value": "test_001_panic_cancelled_by_driver"}, {"name": "host", "value": "WX-AMADRID"}, {"name": "thread", "value": "16000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Xirgo_KB1_Scripts.Panic.test_001_panic_cancelled_by_driver"}]}